agentpluginapi/i_windows_agent_command_builder.py

```diff
@@ -54,11 +54,12 @@ def build_download_command(self, download_options: WindowsDownloadOptions):
         """
 
     @abc.abstractmethod
-    def build_run_command(self, run_options: WindowsRunOptions):
+    def build_run_command(self, run_options: WindowsRunOptions, add_otp=True):
```

Comment:
Why don't you add this to `{Windows,Linux}RunOptions`??

Reply:
Different concerns, I think. One is command build, another is what the command should do. OTP does not describe what the command should do

Reply:
Yes, it does. OTP might be part of the command, so it makes sense if we put it in the options along with other stuff that builds the command.

Reply:
I looked into it from the usage perspective. I think `RunOptions` depend on the target, `add_otp` depends on whether this command will be ran through a script or directly. I don't think it's a big deal either way, so I let you decide

Reply:
I'm still not totally convinced we shouldn't switch to just passing parameters to these methods, but that's not a problem to be solved in this PR.